multiversion/src/main/scala/multiversion/diagnostics/ConflictingTransitiveDependencyDiagnostic.scala

```diff
@@ -26,11 +26,17 @@ class ConflictingTransitiveDependencyDiagnostic(
     def pretty(xs: Iterable[Any]): String =
       if (xs.isEmpty) ""
       else xs.mkString(" ", ", ", "")
+    val depsStr =
+      (foundVersions
+        .map { v =>
+          s"'dependency = ${Dependency(module, v).repr}'"
```

Comment:
Shouldn't it be a colon instead of equals? I think it's not valid YAML with equals.